backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Models/Delegation/DelegationRequestDto.cs

```diff
@@ -18,5 +19,10 @@ public class DelegationRequestDto
         ///     applies to
         /// </summary>
         public string? Action { get; set; }
+
+        /// <summary>
+        ///     The object containing data about the single right delegations
+        /// </summary>
+        public DelegationResponseData? RightDelegationResults { get; set; }
```

Comment:
Where'd find that model? What you mentioned about it shouldn't have Action and Resource twice is correct but those fields should be replcaed with From and To I think.

<img width="346" alt="image" src="https://github.com/Altinn/altinn-access-management-frontend/assets/31471142/10940e2d-8dfa-4d33-bdd2-4908f9de66cf">

Reply:
My screenshot was from the swagger page rendered by the BFF with the current code.
It's the input model for the delegation endpoint with the changes from this PR
What I'm trying to say is that I don't think RightDelegationResults should be part of this input model.
![image](https://github.com/Altinn/altinn-access-management-frontend/assets/22012997/42cbdff7-3cc9-47d6-ab62-ccffb35288e7)

I don't think it should have a From (because Jon Kjetil's model doesn't) but it does already have a To

Reply:
I think we're talking past one another 😅

My picture wasn't from swaggerHub, but from our swagger, generated by the current BFF code.
Here is the picture again:
![image](https://github.com/Altinn/altinn-access-management-frontend/assets/22012997/124a0f99-deef-4046-9bfb-0f554ebc9562)
It is of the model for the _input_ as the model we are discussing, currently named DelegationRequestDTO, is part of the input model. (You can tell because it says Request body above the model)

For reference, this following picture is of the input model in the swaggerHub (so the one the backend will expect)
![image](https://github.com/Altinn/altinn-access-management-frontend/assets/22012997/6ad70e38-0783-4bb1-9010-aca142fdf9a6)

As you can see, these are not equal, but they need to be equal for the BFF-backend communication to work.

What I am saying is that RightDelegationResults should not be part of this _input_ model (which is the one I am discussing in this thread), only of the _output_ model (which is not the one I'm discussing in this thread).

Reply:
Yeah that's correct, I'll remove RightDelegationResults